demo/src/screens/componentScreens/HintsScreen.js

```diff
@@ -10,6 +10,7 @@ export default class HintsScreen extends Component {
       showBottomHint: false,
       showIcon: false,
       targetPosition: 'flex-start',
+      // useTargetFrame: true,
```

Comment:
on Android, all the redioButtons are selected in the screen

Reply:
what do u mean?

Reply:
When I run it in the device all the buttons are marked blue - selected.

Reply:
not sure why, when I run on different emulators it looks good. 
anyway, if anything it's a radioButton issue, let's create a separate ticket to check it.